### PR TITLE
Updated cloud sample for version 4.2

### PR DESCRIPTION
**Pull request by mehmettokgoz**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,10 +2,11 @@ module local/build
 
 go 1.17
 
+require github.com/hazelcast/hazelcast-go-client v1.1.0
```
Comment (yuce) [inline go.mod]: Could you change the dependency to v1.3.0 ?

Comment (yuce) [reply go.mod]: Also, why is `go.mod` different from other code sample PRs?

Comment (mehmettokgoz) [reply go.mod]: Fixed at https://github.com/hazelcast/hazelcast-cloud-go-sample-client/pull/11/commits/785591b859cf7cf5aeece91b341e6b640d4d688f.

**Review by nosan — APPROVED** [review]

**Review by yuce — APPROVED** [review]
Looks good!

**Pull request merged**